src/frontend/components/DashboardVideoPane/index.tsx

```diff
@@ -1,14 +1,15 @@
 import { Box } from 'grommet';
 import React, { useEffect, useState } from 'react';
 import { defineMessages, useIntl } from 'react-intl';
+// eslint-disable-next-line import/no-extraneous-dependencies
 import { Redirect } from 'react-router';
```

Comment:
Hmm this should be better than disable eslint extraneous dependencies error nope?
```suggestion
import { Redirect } from 'react-router-dom';
```